core/src/test/java/org/apache/accumulo/core/file/rfile/RFileTest.java

```diff
@@ -1803,10 +1803,10 @@ public static AccumuloConfiguration setAndGetAccumuloConfig(String cryptoConfSet
 
   public void turnCryptoOnInSiteConfig() {
     SiteConfiguration.clearInstance();
-    SiteConfiguration.getInstance().set(Property.INSTANCE_CRYPTO_SERVICE,
-        AESCryptoService.class.getName());
-    SiteConfiguration.getInstance().set("instance.crypto.opts.kekId", "file:///tmp/testAESFile");
-    SiteConfiguration.getInstance().set("instance.crypto.opts.keyManager", "uri");
+    SiteConfiguration siteConfig = SiteConfiguration.getInstance();
+    siteConfig.set(Property.INSTANCE_CRYPTO_SERVICE, AESCryptoService.class.getName());
```

Comment:
Site configuration should be immutable. It represents the state of a configuration file. We typically put stuff in a "CopyConfiguration" (or is it "ConfigurationCopy"?) to modify it. This setter seems out of place here.

Reply:
Not sure how to fix this. This should be a separate issue anyway. My change does not add the setters. I just reduce calls to getInstance.